app/helpers/application_helper.rb

```diff
@@ -136,7 +136,7 @@ def redirect_back_or_to(options = {}, response_status = {})
 
   def concurrent_events(event)
     return nil unless event.scheduled? && event.program.selected_event_schedules
-    event_schedule = event.program.selected_event_schedules.find_by(event: event)
+    event_schedule = event.program.selected_event_schedules.find { |es| es.event == event }
```

Comment:
Wouldn't we then also need to change it in https://github.com/openSUSE/osem/blob/master/app/views/proposals/show.html.haml#L117 ?